lnrpc/rpc.swagger.json

```diff
@@ -1893,6 +1893,11 @@
           "type": "boolean",
           "format": "boolean",
           "description": "/ Whether this invoice should include routing hints for private channels."
+        },
+        "external_preimage": {
```

Comment:
Just so we know, was this updated from the cli util?

Reply:
yes it was - the only file I updated directly was rpc.proto